node/external/transactionAPI/apiTransactionProcessor.go

```diff
@@ -356,6 +356,18 @@ func (atp *apiTransactionProcessor) fetchTxsForSender(sender string, senderShard
 
 	txsForSender := txCache.GetTransactionsPoolForSender(sender)
 
+	// search user txs for this sender as well
+	// this should return all user txs from user pool
+	userTxsCache := atp.dataPool.UserTransactions().ShardDataStore(cacheId)
+	txCache, ok = userTxsCache.(*txcache.TxCache)
+	if !ok {
+		log.Warn("fetchTxsForSender could not user txs cache cast to TxCache")
```

Comment:
`log.Wan("apiTransactionProcessor.fetchTxsForSender: could not cast relayed inner txs cache to TxCache")`